serverless.yml

```diff
@@ -36,7 +36,6 @@ resources:
       Type: AWS::S3::Bucket
       Properties:
         BucketName: ${self:custom.s3BucketName}
```

Comment:
Current OAI approach is outdated. You can configure recommended OAC approach to connect cloudfront and bucket if you want.